maestro/tests/yamls/agents/dry_run_loop_agent.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: maestro/v1alpha1
+kind: Agent
+metadata:
+  name: generate1-10
+  labels:
+    app: test-example
+spec:
+  model: "llama3.1:latest"
+  description: 
+  tools:
+    - code_interpreter
+    - test
+  instructions: |
+    import random
+    input = random.randint(1, 10)
+
+---
+
+apiVersion: maestro/v1alpha1
+kind: Agent
+metadata:
+  name: countdown
+  labels:
+    app: test-example
+spec:
+  model: "llama3.1:latest"
+  description: this is a test
+  tools:
+    - code_interpreter
+    - test
+  instructions: |
```

Comment:
Shouldn't this be `code` not `instructions`?

Reply:
This is only for the MockAgent.  It uses the `instructions` for processing input and generating the output.